SPECS/jurand/jurand.spec

```
@@ -0,0 +1,86 @@
+Name:           jurand
+Version:        1.3.2
+Release:        4%{?dist}
```

Should be 1 since we are brining it
```suggestion
Release:        1%{?dist}
```